node-fs-readfile/cat.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+
+
+
+
+for (let i=2; i<process.argv.length; i++){
+  const fileName = process.argv[i];
+  fs.readFile(fileName, 'utf8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+  //console.log(fileName);
+}
```

Comment:
These `readFile` executions are asynchronous and running in parallel: there's no guarantee they'll finish in the same order they start.

---

![image](https://user-images.githubusercontent.com/16143771/117342959-ab58a300-ae58-11eb-8bba-c35f7ef5a95d.png)